core/storage/feedback/gae_models.py

```diff
@@ -73,8 +73,24 @@ class GeneralFeedbackThreadModel(base_models.BaseModel):
     # Specifies whether this thread has a related suggestion.
     has_suggestion = ndb.BooleanProperty(
         indexed=True, default=False, required=True)
-    # The number of messages in the thread.
+
+    # The change in status since the last message, if any.
+    updated_status = ndb.StringProperty(choices=STATUS_CHOICES, indexed=True)
+    # Cached value of the number of messages in the thread.
     message_count = ndb.IntegerProperty(indexed=True, default=0)
+    # Cached ID of the last message in the thread.
+    last_message_id = ndb.IntegerProperty(indexed=False, default=None)
```

Comment:
In general I think you can make all these indexed.

You can also omit the "default" arg if it's None; that is what happens in general anyway.

Reply:
Done.